external-crates/move/crates/move-compiler/src/typing/translate.rs

```diff
@@ -4236,20 +4241,29 @@ fn expected_by_name_arg_type(
 fn expand_macro(
     context: &mut core::Context,
     call_loc: Loc,
-    m: ModuleIdent,
+    mod_ident: ModuleIdent,
```

Comment:
To be consistent with other usage in the compiler:
```suggestion
    mident: ModuleIdent,
```

Reply:
Reverted this to the original one now that method name param has a more descriptive name